test/e2e/test_job_submissions.py

```diff
@@ -2015,6 +2016,215 @@ def session_action_has_expected_progress(session_action_id) -> bool:
+        assert attachments is not None
+
+        manifests: list[dict[str, Any]] = attachments["manifests"]
+
+        assert manifests is not None
+        first_manifest = manifests[0]
+
+        input_manifest_path = first_manifest["inputManifestPath"]
+
+        # Find the input manifest
+        queue_job_attachment_settings: dict[str, Any] = deadline_client.get_queue(
+            farmId=deadline_resources.farm.id,
+            queueId=queue_to_use.id,
+        )["jobAttachmentSettings"]
+
+        job_attachments_bucket_name: str = queue_job_attachment_settings["s3BucketName"]
+        root_prefix: str = queue_job_attachment_settings["rootPrefix"]
+
+        s3_client = boto3.client("s3")
+
+        get_manifest_object_result: dict[str, Any] = s3_client.get_object(
+            Bucket=job_attachments_bucket_name,
+            Key=root_prefix + "/Manifests/" + input_manifest_path,
+        )
+
+        get_object_result_body: dict[str, Any] = json.loads(
+            get_manifest_object_result["Body"].read()
+        )
+
+        # Get the Job Attachment bucket file paths of the input files
+        input_file_paths: list[dict[str, Any]] = get_object_result_body["paths"]
+        first_input_file_hash = input_file_paths[0]["hash"]
+
+        # Delete one of the input files from the Job Attachments bucket after confirming that it exists
+
+        s3_client.get_object(
+            Bucket=job_attachments_bucket_name,
+            Key=root_prefix + "/Data/" + first_input_file_hash + ".xxh128",
+        )
+        s3_client.delete_object(
+            Bucket=job_attachments_bucket_name,
+            Key=root_prefix + "/Data/" + first_input_file_hash + ".xxh128",
+        )
+
+        # Start the job, it should fail since one of the input files is missing from the Job Attachments bucket
+
+        deadline_client.update_job(
+            farmId=deadline_resources.farm.id,
+            jobId=job_id,
+            queueId=queue_to_use.id,
+            targetTaskRunStatus="READY",
+        )
+
+        job: Job = Job(
+            farm=deadline_resources.farm,
+            queue=deadline_resources.queue_a,
+            template={},
+            **job_details,
+        )
+        job.wait_until_complete(client=deadline_client)
+
+        # Job should have failed due to not being able to sync attachments
+        assert job.task_run_status == TaskStatus.FAILED
+
+        sessions: list[dict[str, Any]] = deadline_client.list_sessions(
+            farmId=job.farm.id, queueId=job.queue.id, jobId=job.id
+        ).get("sessions")
+
+        found_failed_session_action = False
+        for session in sessions:
+            session_actions = deadline_client.list_session_actions(
+                farmId=job.farm.id,
+                queueId=job.queue.id,
+                jobId=job.id,
+                sessionId=session["sessionId"],
+            ).get("sessionActions")
+
+            logging.info(f"Session actions: {session_actions}")
```

Comment:
```suggestion
            LOG.info(f"Session actions: {session_actions}")
```

Should use same logger as above for consistency. I believe this method goes to [root logger](https://docs.python.org/3/library/logging.html#logging.info)

Reply:
Will address in next PR